content/docs/command-reference/plots/show.md

```diff
@@ -143,15 +143,11 @@ file:///Users/usr/src/dvc_plots/index.html
 We'll use tabular metrics file `logs.csv` for these examples:
 
 ```
-epoch,accuracy,loss,val_accuracy,val_loss
-0,0.9418667,0.19958884770199656,0.9679,0.10217399864746257
-1,0.9763333,0.07896138601688048,0.9768,0.07310650711813942
-2,0.98375,0.05241111190887168,0.9788,0.06665669009438716
-3,0.98801666,0.03681169906261687,0.9781,0.06697812260198989
-4,0.99111664,0.027362171787042946,0.978,0.07385754839298315
-5,0.9932333,0.02069501801203781,0.9771,0.08009233058886166
-6,0.9945,0.017702101902437668,0.9803,0.07830339228538505
-7,0.9954,0.01396906608727198,0.9802,0.07247738889862157
+epoch,loss,accuracy
```

Comment:
Seems like the definition, synopsis, description in this file need an update too?